mmhuman3d/data/data_converters/humman.py

```diff
@@ -0,0 +1,277 @@
+import glob
+import os
+
+import numpy as np
+from tqdm import tqdm
+
+from mmhuman3d.core.conventions.keypoints_mapping import (
+    convert_kps,
+    get_keypoint_idx,
+    get_keypoint_num,
+)
+from mmhuman3d.data.data_structures import SMCReader
+from mmhuman3d.data.data_structures.human_data import HumanData
+from .base_converter import BaseModeConverter
+from .builder import DATA_CONVERTERS
+
+
+@DATA_CONVERTERS.register_module()
+class HuMManConverter(BaseModeConverter):
+    """A mysterious dataset that will be announced soon."""
+
+    skip_no_iphone = True
+    skip_no_keypoints3d = True
+    downsample_ratio = 10  # uniformly select 1 from every 10 examples
+
+    keypoint_convention = 'coco_wholebody'
+    left_hip_idx = get_keypoint_idx('left_hip', keypoint_convention)
+    right_hip_idx = get_keypoint_idx('right_hip', keypoint_convention)
+
+    ACCEPTED_MODES = ['test', 'train']
+
+    def _make_human_data(
+        self,
+        mode,
+        smpl,
+        keypoints_convention,
+        image_path,
+        image_id,
+        bbox_xywh,
+        keypoints_2d,
+        keypoints2d_mask,
+        keypoints_3d,
+        keypoints3d_mask,
+    ):
+        # use HumanData to store all data
+        human_data = HumanData()
+
+        # downsample idx
+        if mode == 'train':
+            select = [
```

Comment:
`selected_idxs` or `idxs`?

Reply:
`selected_inds`, fixed in https://github.com/open-mmlab/mmhuman3d/pull/73/commits/f943649f4a1fd45aa0e9e7134989f0b4c127b31e.